apps/frontend-pwa/.env.assessment.stg

```
@@ -0,0 +1,29 @@
+COOKIE_DOMAIN=.klicker.stg.df-app.ch
+API_DOMAIN=assessment-api$COOKIE_DOMAIN
+
+NEXT_PUBLIC_AVATAR_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/avatars"
+NEXT_PUBLIC_IMAGE_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/img"
+NEXT_PUBLIC_ADD_RESPONSE_URL="https://response-api-assessment$COOKIE_DOMAIN/AddResponse"
+NEXT_PUBLIC_API_URL="https://$API_DOMAIN/api/graphql"
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_URL key should go before the NEXT_PUBLIC_AVATAR_BASE_PATH key

(UnorderedKey)